influx_data.sh

```diff
@@ -208,6 +208,83 @@ _blocks_fill_90='from(bucket: "tds")|> range(start:'${start_time}' ,stop:'${stop
   				|> aggregateWindow(every: '${window_interval}',  fn: (column, tables=<-) => tables |>  count(column: "bank_slot"))
     			|> sum(column: "bank_slot")
 				|> drop(columns: ["_start", "_stop"])'
+#skip_rate
+_mean_skip_rate='data_max=from(bucket: "tds")|> range(start:'${start_time}' ,stop:'${stop_time}')
+				|> filter(fn: (r) => r["_measurement"] == "bank-new_from_parent-heights")
+				|> filter(fn: (r) => r["_field"] == "slot" or r["_field"] == "block_height")
+				|> aggregateWindow(every:'${oversize_window}', fn:max)
```

Comment:
looks like a most of the code is repeated. I wonder if it is possible in bash to do a function for these requests, so when later you will need to change the general query schema it will end up in one change instead of N

Reply:
Do you mean skip_rate part of code and maybe queries with the same section
Yes, it is good to make a function in the future.

Reply:
right, so basically make a function for such queries